src/js/main.js

```diff
@@ -744,14 +743,11 @@ function getConfiguratorVersion() {
 }
 
 function updateTopBarVersion(firmwareVersion, firmwareId, hardwareId) {
-
-    const configuratorVersion = getConfiguratorVersion();
-    const firmwareVersionAndId = getFirmwareVersion(firmwareVersion, firmwareId);
-    const targetVersion = getTargetVersion(hardwareId);
-
-    const versionText = `${configuratorVersion}<br />${firmwareVersionAndId}<br />${targetVersion}`;
-
-    $('#logo .logo_text, #tab_logoversion .version').html(versionText);
+    // Vue migration START
+    window.vm.firmwareVersion = firmwareVersion;
+    window.vm.firmwareId = firmwareId;
+    window.vm.hardwareId = hardwareId;
+    // Vue migration END
```

Comment:
Only to understand how it will work in the future: I suppose any element will have it's own variable to react with. So we will have a lot of new variables inside `window.vm`, true?

Reply:
These serve as a proxy objects, so we would have more of these. But they pretty much replace the jquery usage where jquery was manipulating the DOM directly you would just update the objects.

Some of the global objects can be just passed as they are as well.

In general I was thinking on inside out approach. Start with small things like logo. Then once enough of widgets are implemented in the same section, merge them under single Vue instance.